yt_dlp/extractor/adobepass.py

```diff
@@ -1526,6 +1528,27 @@ def extract_redirect_url(html, url=None, fatal=False):
                         if 'Please try again.' in saml_login_page:
                             raise ExtractorError(
                                 'We\'re sorry, but either the User ID or Password entered is not correct.')
+                    else:
+                        # elif 'Please wait ...' in provider_redirect_page and "'N'== \"Y\"" in provider_redirect_page:
```

Comment:
?

Reply:
Removed since it's unused, it was added there by the original author.